sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -3313,6 +4578,22 @@ public partial class RestServiceLinkedService : Azure.Analytics.Synapse.Artifact
         public object Url { get { throw null; } set { } }
         public object UserName { get { throw null; } set { } }
     }
+    public partial class RestSource : Azure.Analytics.Synapse.Artifacts.Models.CopySource
+    {
+        public RestSource() { }
+        public object AdditionalHeaders { get { throw null; } set { } }
+        public object HttpRequestTimeout { get { throw null; } set { } }
+        public object PaginationRules { get { throw null; } set { } }
+        public object RequestBody { get { throw null; } set { } }
+        public object RequestInterval { get { throw null; } set { } }
+        public object RequestMethod { get { throw null; } set { } }
+    }
+    public partial class RetryPolicy
+    {
+        public RetryPolicy() { }
+        public object Count { get { throw null; } set { } }
+        public int? IntervalInSeconds { get { throw null; } set { } }
+    }
```

Comment:
This is another one of those things like Pipeline that's super overloaded in the REST client space and we should think about whether there's a better name we can use to make more Synapse specific in the future.

Reply:
Sure, we will fix this in future